elastic4s-core/src/main/scala/com/sksamuel/elastic4s/mappings/MappingDefinition.scala

```diff
@@ -41,13 +45,15 @@ case class PutMappingDefinition(indexesAndType: IndexesAndType,
                                 routing: Option[RoutingDefinition] = None,
                                 timestamp: Option[TimestampDefinition] = None,
                                 templates: Seq[DynamicTemplateDefinition] = Nil,
-                                id: Option[IdField] = None
+                                id: Option[IdField] = None,
```

Comment:
I tried it in the test to check if the request get rejected or something and it seems that this parameter won't be reflected in the Put body at all. So I will remove it.

Thank you for the comment :)